CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## Unreleased
+
+### Changed
+
+- Shorter form of `composer require` command example in `README.md`
+
```

Comment:
```suggestion
```
There is no sense to  add `README` fixes into changelog

Reply:
Thanks @jetexe!